src/main/java/com/o19s/es/ltr/query/StoredLtrQueryBuilder.java

```diff
@@ -157,14 +163,24 @@ private static void validateActiveFeatures(FeatureSet features, LtrQueryContext
 
     @Override
     protected RankerQuery doToQuery(QueryShardContext context) throws IOException {
+        this.ltrStats.getStat(StatName.LTR_REQUEST_TOTAL_COUNT.getName()).increment();
+        try {
+            return _doToQuery(context);
+        } catch (Exception e) {
+            ltrStats.getStat(StatName.LTR_REQUEST_ERROR_COUNT.getName()).increment();
+            throw e;
+        }
+    }
+
+    private RankerQuery _doToQuery(QueryShardContext context) throws IOException {
```

Comment:
`_doToQuery` not a common java naming convention. Suggest change to other name like `createRankerQuery`